extensions/amp-consent/amp-consent.md

```diff
@@ -193,6 +193,33 @@ The prompt UI is defined within the consent instance config. The `promptUI` attr
 
 AMP displays prompt UI on page load or by user interaction. The prompt UI is hidden based on the three user actions described below.
 
+#### Prompt UI for Stories
+
+The `amp-story` extension provides a [default prompt UI](https://user-images.githubusercontent.com/1492044/40135514-8ab56d10-5913-11e8-95a2-72ac01ff31e0.png), that requires using a `<amp-story-consent>` component as the prompt UI. This component content (`title`, `message`, and a list of `vendors`) has to be specified in the main consent configuration.
+
+*Example*: Displays a prompt user interface on an AMP Story
+
+```html
+<amp-consent layout="nodisplay" id="consent-element">
+  <script type="application/json">
+  {
+    "consents": {
+      "my-consent": {
+        "checkConsentHref": "https://foo.com/api/show-consent",
+        "promptUI": "consent-ui"
+      }
+    },
+    “story-consent”: {
```

Comment:
@gmajoulet I know it's kind of late to raise this. But why are we putting the config in `<amp-consent>`? not in `<amp-story-consent>` How's the decision made?

Reply:
We thought it would be easier to piggyback on the ``<amp-consent>`` config for two reasons:
- Less boilerplate for publishers, and easier to use since all there's to do is adding a ``story-consent`` key to the existing config. (My personal thought on this is that ``amp-consent`` is one component, that happens to come with a default UI on stories. Two configs for one component/feature can be less straightforward than one single "consent config JSON")
- Allows us to piggyback on the existing validator rules, plus validation done by ``amp-consent``

If you feel strongly about this, it's a bit late but I imagine we can still consider changing.

Reply:
Honestly I don't like this. When we designed `<amp-consent>` it consists two parts. One consent config that handles consent storage, consent blocking behavior. This is part in the json config. And the other part is consent UI component. It needs to live inside the `<amp-consent>` only because they can pop up without user interaction, and we need it to be wrapped within the position fixed `<amp-consent>` element. 

To me the `"story-consent"` here is only prompted UI related, and I don't know if it should lives within the consent json config. 

However `“vendors”: ["Item 1", "Item 2", "Item 3", "Item 4"]` is consent related, is there any requirement to store consent status for those vendors individually in the future?